docker/Dockerfile

```diff
@@ -7,16 +7,19 @@ RUN apt-get update && \
 
 WORKDIR /usr/src/app
 
-COPY docker/package.json .
+# install npm packages
+COPY docker/package.json ./
 RUN npm install
 
-# Bundle app source
-COPY build/ ./build
+# copy our code
 COPY docker/server ./server
-COPY start.sh .
+COPY public ./public
+COPY img ./img
+COPY src ./src
+COPY build ./build
+COPY ./start.sh ./
```

Comment:
Does this file need to exist at the root? Why not move it into the `docker/` directory? Then this line becomes:

```
COPY ./docker/start.sh ./
```

Reply:
yep. will move.